utilities/run_unit_tests.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 
 usage () {
     echo -e "\tUSAGE: $0 <main|initial> <device|simulator>"
```

Comment:
Are we planning to change invocation of this script as well?

Reply:
bash is more suitable. sh has some missing features.

Reply:
for example, in `${OPTARG^}` the casing operation (`^`) is absent in `sh`.

Reply:
I mean using -c/-f/-t flags wise

Reply:
i misinterpreted. will update.